src/components/dashboard/FaceVerification/screens/ErrorScreen.js

```diff
@@ -49,17 +53,41 @@ const ErrorScreen = ({ styles, screenProps }) => {
   }, [kindOfTheIssue, resetAttempts])
 
   useEffect(() => {
-    if (!isGeneralError) {
+    // tracking attempt here as we should track only "general" error
+    // (when "something went wrong on our side")
+    // if there will be a Human errors (like DeviceOrientation or Permission errors)
+    // it will be skip and do not consider as failed attempt
+    if (!isGeneralError && kindOfTheIssue !== 'DuplicateFoundError') {
```

Comment:
the comment says  only "something  wrong on our side" is incorrect
like  explained it should track ALL failures that happen after user started the  FV process
have you verified it does that? where does the "retry" screen trigger the tracking attempt? the retry screen is not part of our error screens, its part of  the Zoom UI

Reply:
@sirpy If some error will occur during the FaceVerification process (even Zoom errors) it will be directed/passed to the ErrorScreen component and there we will track it as a failed attempt.
We are capturing all of the errors that could occur during the FV process.
I already tested the case when face verification started but the face wasn't captured in 1 minute - so the zoom generates timeout error which was successfully tracked as failure attempt.

Reply:
@sirpy 
@yaroslav-fedyshyn-nordwhale 
i've added tracking of all exceptions having 'retry' option (including ZoOm's retry screen).
How all them will be sent once user reached 'something went wrong our side' screen